packages/client/hmi-client/src/workflow/ops/document/tera-document-operation-node.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+	<main v-if="!fetchingDocument">
+		<template v-if="document">
+			<div>{{ document?.name }}</div>
```

Comment:
Look at other node operator header
```suggestion
			<h5><strong>{{ document?.name }}</strong></h5>
```